elements/pfe-icon/demo/index.html

```diff
@@ -329,21 +329,41 @@ <h2>web icons</h2>
     <pfe-icon size="md" icon="web-youtube2"></pfe-icon>
     <pfe-icon size="md" icon="web-youtube"></pfe-icon>
 
+    <h2>Circled icons</h2>
+    <pfe-icon size="xl" circled="complement" icon="rh-aed"></pfe-icon>
+    <pfe-icon size="xl" circled="accent" icon="rh-alert-downtime"></pfe-icon>
+    <pfe-icon size="xl" circled="darkest" icon="rh-api"></pfe-icon>
+    <pfe-icon size="xl" circled="dark" icon="rh-app-b"></pfe-icon>
+    <pfe-icon size="xl" circled icon="rh-application-alt"></pfe-icon>
+    <pfe-icon size="xl" circled="light" icon="rh-application-mobile"></pfe-icon>
+    <pfe-icon size="xl" circled="lightest" icon="rh-application"></pfe-icon>
+
+    <h2>Circled icons: forced stretch</h2>
+    <div class="pfe-l-grid pfe-m-gutters pfe-m-all-6-on-xs pfe-m-all-3-col">
+      <pfe-icon size="xl" circled="complement" icon="rh-aed"></pfe-icon>
+      <pfe-icon size="lg" circled="accent" icon="rh-alert-downtime"></pfe-icon>
+      <pfe-icon size="md" circled="darkest" icon="rh-api"></pfe-icon>
+      <pfe-icon size="sm" circled="dark" icon="rh-app-b"></pfe-icon>
+      <pfe-icon size="2x" circled icon="rh-application-alt"></pfe-icon>
+      <pfe-icon size="3x" circled="light" icon="rh-application-mobile"></pfe-icon>
+      <pfe-icon size="4x" circled="lightest" icon="rh-application"></pfe-icon>
```

Comment:
doesnt have to be this MR but these should be pfe prefixed, is there a different issue for this?

Reply:
Yeah, different issue though, I wasn't going to squirrel on this branch too much.